docs/development/module/fields.md

```diff
@@ -140,8 +212,8 @@ dependencies:
   enforced:
     module:
       - my_module
-id: monitor
-label: Monitor
+id: organic
+label: Organic
 entity_types: null
 ```
 
```

Comment:
Great Job,
Perhaps an additional comment on id in file name should match the `id `in the file `farm_flag.flag.[id].yml` ?

Reply:
I've added an additional comment. Took an educated guess that the file name is generally in the form `[select_module_name].[select_field].[id].yml`, so hopefully that is close enough :).